### DISCUSSION TITLE
Upload an existing project ?

### DISCUSSION THREAD

**Discussion by cassepipe** [SvelteLab/SvelteLab#174] [Ideas]
First, whoever made this tool, thank you. I am learning SvelteKit and I have been looking for a too like this one as the Svelte official REPL is too limited. 

I have made a small and focused example from my project on my machine and I would like to upload it to SvelteLab instead of having to copy paste each file.
I don't see an Upload command in the command palette not is it mentionned in this repo's Readme.

Is there a way to do this ?

If not, can we have it ? I would greatly increase the shareability of SvelteKit example projects.

I suspect the reason there isn't one may be hosting costs, maybe there could be upload maximum size ? A expiry date for examples ? Premium accounts for people who want to host their example wihtout an expiry date ?

**Comment by paoloricciuti** [comment]
Yeah we plan for sure to allow for uploads but as you've said that's a tricky thing to get right. One thing that we plan to do in the immediate future is to allow the user to open a GitHub repo inside SvelteLab

**Comment by SrGeneroso** [comment]
You can now!!
[https://sveltelab.dev/github.com/username/repo](https://docs.sveltelab.dev/sharing-and-saving#open-project-from-github)
